storage/innobase/include/sync0policy.h

```diff
@@ -50,7 +50,7 @@ class MutexDebug {
 			m_mutex(),
 			m_filename(),
 			m_line(),
-			m_thread_id(os_thread_id_t(ULINT_UNDEFINED))
+			m_thread_id((ulint)os_thread_id_t(ULINT_UNDEFINED))
```

Comment:
That is cast `ULINT_UNDEFINED` to `os_thread_id_t`and then cast it back to `uliint`? :)

Please use just ULINT_UNDEFINED here.